src/containers/safari/tags.state.js

```diff
@@ -118,20 +118,30 @@ export const tagsSagas = [
 
 /* SAGAS :: SELECTORS
 –––––––––––––––––––––––––––––––––––––––––––––––––– */
-const getTaggingPayload = state => {
+const getTaggingState = state => {
+  const { suggested, used } = state.tags
+
   return {
     item_id: state.saves.item_id,
-    tags: state.tags.used
+    used,
+    suggested
   }
 }
 
 /* SAGAS :: RESPONDERS
 –––––––––––––––––––––––––––––––––––––––––––––––––– */
 function* tagChanges() {
   yield delay(1000)
-  const payload = yield select(getTaggingPayload)
-
-  if (payload) {
+  const taggingState = yield select(getTaggingState)
+  const { used, suggested, item_id } = taggingState
+  const usedSuggested = used.filter(usedTag => suggested.includes(usedTag))
+  const payload = {
+    item_id,
+    tags: used,
+    suggestedCount: suggested.length,
+    usedSuggestedCount: usedSuggested.length
+  }
+  if (taggingState) {
```

Comment:
We should be checking that there are tags we are sending through.  Pretty sure this will always return true in its current state.  Something like `if(used.length)`.  Looks like the previous code was off as well.